sql/tsql/TSqlLexer.g4

```diff
@@ -899,6 +918,12 @@ BINARY:              '0' 'X' HEX_DIGIT*;
 FLOAT:               DEC_DOT_DEC;
 REAL:                (DECIMAL | DEC_DOT_DEC) ('E' [+-]? DEC_DIGIT+);
 
+IPV4_OCTET:          (DEC_DIGIT DEC_DIGIT?) | ([0-2] DEC_DIGIT DEC_DIGIT);
```

Comment:
Excess parentheses. Also, I guess this rule should be marked with `fragment`, see I guess it should be fragment, see https://github.com/antlr/antlr4/blob/master/doc/lexer-rules.md#lexer-rules

```suggestion
fragment IPV4_OCTET:          DEC_DIGIT DEC_DIGIT? | [0-2] DEC_DIGIT DEC_DIGIT;
```